src/TestCase.php

```diff
@@ -198,4 +198,26 @@ protected function getEnvironmentSetUp($app)
     {
         // Define your environment setup.
     }
+
+    /**
+     * Migrate Laravel's default migrations.
+     * 
+     * @param  string $database
+     * 
+     * @return void
+     */
+    public function runLaravelDefaultMigrations($database = null)
+    {
+        $options = ['--path' => '../fixture/migrations'];
```

Comment:
Tried it and yes, that works as well. Will change it.
However, it does not solve the `cannot redeclare class` problem, which a rename of `tests/migrations/2013_07_26_182750_create_users_table.php` would.